### PR TITLE
SEO-181471-ASPNET-HTTP-TO-HTTPS

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
I have fixed the http issues, please review.
Regards,
Scholar.

### CODE REVIEW

**Review by MallikaRK — CHANGES_REQUESTED** [review]
commented

File: aspnet/TreeView/Checkboxes.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Checkboxes in TreeView | TreeView | ASP.NET | Syncfusion
-description: Specify checkboxes in TreeView and its settings
+description: Learn here about specifying checkboxes in Syncfusion ASPNET TreeView control, its settings and more.
```
[the remark this answers is not in the document]
Comment (Scholar-jaruha) [reply aspnet/TreeView/Checkboxes.md]: changed

**Review by MallikaRK — APPROVED** [review]
approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**